build_tools/kokoro/gcp_ubuntu/cmake/linux/x86-swiftshader-asan/build.sh

```diff
@@ -102,15 +102,6 @@ label_exclude_regex="($(IFS="|" ; echo "${label_exclude_args[*]?}"))"
 declare -a excluded_tests=(
   "iree/base/internal/file_io_test"
   "bindings/tflite/smoke_test"
-  "iree/hal/cts/allocator_test"
-  "iree/hal/cts/buffer_mapping_test"
-  "iree/hal/cts/command_buffer_test"
-  "iree/hal/cts/descriptor_set_layout_test"
-  "iree/hal/cts/driver_test"
-  "iree/hal/cts/event_test"
-  "iree/hal/cts/executable_layout_test"
-  "iree/hal/cts/semaphore_submission_test"
-  "iree/hal/cts/semaphore_test"
```

Comment:
Expecting test failures here and the test names changed.

I'm also going to check the total test execution times before/after this change for regressions.